internal/ethapi/api_test.go

```diff
@@ -157,3 +169,21 @@ func allTransactionTypes(addr common.Address, config *params.ChainConfig) []type
 		},
 	}
 }
+
+func TestNewPersonalAccount(t *testing.T) {
+	d := t.TempDir()
+	ks := keystore.NewKeyStore(d, keystore.StandardScryptN, keystore.StandardScryptP)
+	b := &backendTest{
+		backends: []accounts.Backend{ks},
+	}
+	nonceLock := new(AddrLocker)
+	pa := NewPersonalAccountAPI(b, nonceLock)
+	addr, err := pa.NewAccount("password")
+	if err != nil {
+		t.Fatalf("failed to create account: %v", err)
+	}
+
+	if !common.IsHexAddress(addr.String()) {
+		t.Fatalf("address isn't hex encoded: %s", addr)
+	}
+}
```

Comment:
I'm not sure this test really brings anything new? All it does it tests

1. That an account can be created
2. That the response is hex-encoded

Unless I missed something, might aswell remove it

Reply:
Yes, that is exactly what it does. 

I saw that this part of the codebase didn't have much unit testing and took it as an opportunity to add a couple. 

I can remove it if you feel is superficial!

Reply:
Sorry for late response. Yes, please remove it, IMO it doesn't add anything. Since you are using `NewPersonalAccountApi` (not going via any url dispatcher), you are guaranteed to obtain a `common.AddressEIP55`. 
The check `common.IsHex..` is basically a long-winded unit-test, and `IsHex` doesn't even check anything related to EIP-55.

Reply:
Yes, that makes sense while it helped me validate that this actually works initially, I can see it doesn't add a lot in terms of testing. Reverted!